src/classes/filter/EngineLevelFilter.as

```diff
@@ -255,6 +259,22 @@ package classes.filter
             return false;
         }
 
+        /**
+         * Compares 2 Number values with a greater than comparison, unless inverted.
+         * @param	value1	Input Value
+         * @param	value2	Value to compare to.
+         * @param	inverse	Use inverse comparisons.
```

Comment:
Missing param or remove this.

Reply:
That's the IS/ISN'T (inverse) operator, it's not passed into the function but it's still referenced (see compareString() a little bit further down)

Reply:
Oh okok. I usually reserve the @param docstring to actual parameters.

Reply:
Honestly doesn't make a difference to me either way -- was just following the format that was already used haha.